src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int smallerSide;
+    private final int biggerSide;
+    private final int height;
+
+    public IsoscelesTrapezoid(String color, int smallerSide, int biggerSide, int height) {
+        super(color);
+        this.smallerSide = smallerSide;
+        this.biggerSide = biggerSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (smallerSide + biggerSide) * height * 0.5;
+    }
+
+    @Override
+    public void toDrawFigure() {
+        System.out.println("Figure: isosceles trapezoid, area: " + getArea() + " sq. units, "
+                + "smaller side: " + smallerSide + " units, bigger side: " + biggerSide
+                + " units, height: " + height + " units, color: " + color);
+    }
```

Comment:
The method name 'toDrawFigure' does not follow the Java naming conventions for methods. It should be a verb or a verb phrase, so a better name would be 'drawFigure' as per the checklist item about not using verbs for class/interface names.